docs/util/stack/stack.md

```diff
@@ -0,0 +1,77 @@
+---
+parent: Util
+layout: default
+title: Stack
+nav_order: 3
+---
+
+<!-- KDoc -->
+[StackKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/index.html
+[peekKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/peek.html
+[peekOrNullKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/peek-or-null.html
+[peekAllKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/peek-all.html
+[pushKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/push.html
+[pushAllKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/push-all.html
+[popKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/pop.html
+[popOrNullKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/pop-or-null.html
+[popAllKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/pop-all.html
+[sizeKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/index.html
+[isEmptyKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/is-empty.html
+[isNotEmptyKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/is-not-empty.html
+[clearKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/clear.html
+[sortKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/sort.html
+[shuffleKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/shuffle.html
+[indexOfKDoc]: https://tudo-aqua.github.io/bgw/kotlin-docs/bgw-core/tools.aqua.bgw.util/-stack/index-of.html
+
+[NoSuchElementExceptionKDoc]: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-no-such-element-exception/
+
+<!-- Start Page -->
+# Stack
+
+{: .no_toc}
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+
+## Usage
+The [Stack][StackKDoc] class is a kotlin implementation of the common stack collection.
```

Comment:
Could specify that this is a utility class, since stacks are a common data structure in board games.